### ISSUE TITLE
[FR] Display the datetime of last (successful) repos sync

### ISSUE THREAD

**Issue by breversa** [NeoApplications/Neo-Store#511]
Currently, there is no way to see when a particular (or all) repository(/ies) was successfully synced, leading (me) to uselessly and repeatedly tapping the "sync" button "just to be sure".

Having a "last full (= all repos correctly synced) sync on 2023-08-25 10:48" and "last partial (= at least one repo correctly synced) sync on 2023-08-25 10:48" visibly displayed (or easily accessible) would help a lot.

Maybe in a smaller font below "Neo Store" on the top-left of the screen?

The last sync should also be displayed in the repos list, for each of them:
- last time the repo was synced
- last time the repo had an actual content change

This would help spot faulty or no longer updated repos.

**Comment by machiav3lli** [comment]
The info is available in each repo, just open their sheet to see the time of last successful update (sync with content change). I'm not sure to how extent what you're suggesting could be ambigous, as I understand the goal, but I still don't see a logical, compact way to do that in the main UI, but in the repos page, I guess a small text can be added to all repo items..

**Comment by breversa** [comment]
You’re correct that the latest repo change is displayed in each *active* repo. For some reason I checked an inactive one… 🤦‍♂️ 

Would it be possible to display "Repos were last successfully synced on the 2023-08-25 10:48" in the collapsible top row, below "Neo Store" ?, so that it slips out of the way when scrolling down?

**Comment by machiav3lli** [comment]
The collapsible top row is not collapsible since couple of alphas, but maybe I'll add a long press action to the sync button to show this info.

**Comment by machiav3lli** [comment]
Already added since 1.0.8 (or earlier)

**Comment by machiav3lli** [comment]
And some were complimented in 1.1.0-beta01
